From: Anonymous <noreply@weblate.org>
Date: Tue, 25 Aug 2026 07:50:57 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (7 of 7 strings)

Translation: openkylin-huanghe/kylin-virtual-keyboard
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-virtual-keyboard/ko_KP/
---
 translations/translation_ko_KP.ts | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/translations/translation_ko_KP.ts b/translations/translation_ko_KP.ts
index 68f3603..4cb8997 100644
--- a/translations/translation_ko_KP.ts
+++ b/translations/translation_ko_KP.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../src/virtualkeyboardentry/floatbutton.cpp" line="142"/>
         <source>Click to show virtual keyboard</source>
-        <translation type="unfinished"></translation>
+        <translation>가상 키보드 표시</translation>
     </message>
 </context>
 <context>
@@ -14,7 +14,7 @@
     <message>
         <location filename="../qml/HideButton.qml" line="38"/>
         <source>close</source>
-        <translation type="unfinished"></translation>
+        <translation>닫기</translation>
     </message>
 </context>
 <context>
@@ -23,12 +23,12 @@
         <location filename="../qml/PlacementModeButton.qml" line="40"/>
         <location filename="../qml/PlacementModeButton.qml" line="67"/>
         <source>Floating Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>플로팅 상태</translation>
     </message>
     <message>
         <location filename="../qml/PlacementModeButton.qml" line="78"/>
         <source>Docking Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>도킹 상태</translation>
     </message>
 </context>
 <context>
@@ -36,12 +36,12 @@
     <message>
         <location filename="../src/virtualkeyboardentry/virtualkeyboardentrymanager.cpp" line="83"/>
         <source>Disable the float button</source>
-        <translation type="unfinished"></translation>
+        <translation>키보드 플로팅 버튼 비활성화</translation>
     </message>
     <message>
         <location filename="../src/virtualkeyboardentry/virtualkeyboardentrymanager.cpp" line="93"/>
         <source>Enable the float button</source>
-        <translation type="unfinished"></translation>
+        <translation>키보드 플로팅 버튼 활성화</translation>
     </message>
 </context>
 <context>
@@ -49,7 +49,7 @@
     <message>
         <location filename="../src/virtualkeyboardentry/virtualkeyboardtrayicon.cpp" line="41"/>
         <source>Virtual Keyboard</source>
-        <translation type="unfinished"></translation>
+        <translation>가상 키보드</translation>
     </message>
 </context>
 </TS>
